tools/bazel.rc

```diff
@@ -12,6 +12,8 @@ build:opt --copt=-Wframe-larger-than=16384
 build:dbg --compilation_mode=dbg
 
 build:asan --strip=never
+build:asan --copt -Wno-macro-redefined
```

Comment:
why is this needed?

Reply:
This is to suppress macro redefined warnings. `-D_FORTIFY_SOURCE=1` enabled by default on Mac OS.